network/loop_test.go

```diff
@@ -133,11 +133,12 @@ func TestTLSConnection(t *testing.T) {
 			require.NotNil(t, l, "newLoop should not return nil for valid TLS config")
 
 			// Create a test series for sending
-			l.series = append(l.series, createSeries(t))
+			pending := make([]types.MetricDatum, 1)
```

Comment:
I think this should be just as performant and more clear.

```suggestion
			pending := []types.MetricDatum {
						createSeries(t),
			}
```